docs/spec/markdown/patches.md

```diff
@@ -85,11 +85,12 @@ The `remove-public-keys` _Patch Action_ describes the removal of cryptographic k
 
 #### `ietf-json-patch`
 
-::: warning
-Implementers of Sidetree-based DID networks that are open, permissionless, and decentralized should carefully consider the ramifications of adding free-form _Patch Actions_ that are not strictly evaluated by nodes, as this is very likely to result in abuse of the implementation in a myriad of ways that are not intended, and could compromise the game theoretical economic viability of the implementation.
-:::
+The `ietf-json-patch` _Patch Action_ describes an update to a DID Document, using [IETF JSON Patch](https://tools.ietf.org/html/rfc6902). To construct a `ietf-json-patch` _Patch Action_, compose an object as follows:
```

Comment:
Suggested language to make this a bit more general for Methods that may choose to patch against in intermediate, abstract representation of DID state:

"The `ietf-json-patch` _Patch Action_ describes an update to a DID's state, using [IETF JSON Patch](https://tools.ietf.org/html/rfc6902)."

Reply:
@csuwildcat can you make this a suggestion that I can actually approve and merge?